src/Mapbender/CoreBundle/Resources/public/mapbender.element.searchRouter.js

```diff
@@ -526,70 +563,55 @@
          * @param  jQuery.Event event Mouse event
          */
         _resultCallback: function(event){
-            var widget = this;
-            var options = widget.options;
-            var routes = options.routes;
-            var row = $(event.currentTarget),
-                feature = $.extend({}, row.data('feature').getFeature()),
-                map = feature.layer.map,
-                callbackConf = routes[widget.selected].results.callback,
-                srs = Mapbender.Model.getProj(widget.searchModel.get("srs"));
-            var mapProj = Mapbender.Model.getCurrentProj();
-            if(srs.projCode !== mapProj.projCode) {
-                feature.geometry = feature.geometry.transform(srs, mapProj);
+            var feature = $(event.currentTarget).data('feature').getFeature().clone();
+            var srs = Mapbender.Model.getProj('EPSG:' + feature.attributes.srid);
+            var results = this.options.routes[this.selected].results;
+            var c_options = results.callback && results.callback.options ? results.callback.options : null;
```

Comment:
@psch, please names variables properly!
Property names SHOULD NOT be prefixed with a single underscore to indicate protected or private visibility.

A property declaration looks like the following: "callBackOptions" or just  "options" if it's inside of "_resultCallback" method.